### PR TITLE
Updating documentation about linters.

### PR DESCRIPTION
**Pull request by sweeneydavidj**
Fixes: Consolidation and updating of documentation about linters used in the project.

**Changes proposed in this pull request:**
- Removed outdated linter information , about SCSS, from main README
- Removed outdated information about Ember Suave etc from top of CONTRIBUTING guide.
- Added to existing documentation in Linter section of CONTRIBUTING guide - including external links to documentation about eslint-plugin-ember-suave and ember-template-lint.

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -124,8 +124,9 @@ In addition, linters can help to enforce coding standards, find unused variables
 
 **HospitalRun** uses the following linters:
 
-1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/).
+1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/). In addition to the default set of rules ESLint is setup to use eslint-plugin-ember-suave you can find more information about that [here](https://github.com/DockYard/eslint-plugin-ember-suave).
```
Comment (jkleinsc) [inline .github/CONTRIBUTING.md]: We need a link here to Dockyard's Ember.js style guide:
https://github.com/DockYard/styleguides/blob/master/engineering/ember.md

This has more helpful information than the  eslint-plugin-ember-suave page.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -124,8 +124,9 @@ In addition, linters can help to enforce coding standards, find unused variables
...
 2. [stylelint](http://stylelint.io/) for Stylesheets. You can find stylelint User guide [here](http://stylelint.io/user-guide/).
```
Comment (jkleinsc) [inline .github/CONTRIBUTING.md]: Can you correct the grammar here to You can find _the_ stylelint user guide here

**Comment by sweeneydavidj** [comment]
@jkleinsc thanks for the review! I've added in the changes that you mentioned.

**Comment by sweeneydavidj** [comment]
@jkleinsc I've added in those changes, hopefully that reads better now.

**Pull request merged**